Update set_up_an_account.adoc (#998)
starknet-io/starknet-docs: components/Starknet/modules/quick_start/pages/set_up_an_account.adoc

[source,shell]
----
starkli account fetch <SMART_WALLET_ADDRESS> --output ~/.starkli-wallets/deployer/account.json --rpc <YOUR_RPC_ENDPOINT_HERE>
----

> Starkli is centric around JSON-RPC, and the JSON-RPC provider is considered canonical. Users are strongly recommended to use JSON-RPC. There are a few options to obtain access to a JSON-RPC endpoint:

Reviewer: The Feeder Gateway is deprecated, and much support will be removed soon, so I wouldn't even mention JSON-RPC here as an option, because there really isn't an alternative.
```suggestion
You can obtain access to a JSON-RPC endpoint in one of the following ways:
```

> - hosting your own node with pathfinder or juno; or

Reviewer: ```suggestion
- Host your own node with Pathfinder, Juno, or Papyrus.
```

> - using a third-party JSON-RPC API provider like Infura, Alchemy, Chainstack, or Nethermind.

Reviewer: ```suggestion
- Use a third-party JSON-RPC API provider like Infura, Alchemy, Chainstack, or Nethermind.
```
